src/redux/books/apiActions.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { addBook, fetchError } from './books';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps';
+const appId = 'ux6EkcA2ZXp2cRIPwiAL';
+const baseUrl = axios.create({ baseURL: `${url}/${appId}` });
+export const getBookApi = () => (dispatch) => {
+  baseUrl.get('/books').then((res) => {
+    dispatch(addBook(res.data));
+  }).catch(() => {
+    dispatch(fetchError());
+  });
+};
+
+export const removeBookApi = (ID) => (dispatch) => {
+  baseUrl.delete(`/books/${ID}`, { item_id: ID }).then(() => {
+    dispatch(getBookApi());
+  }).catch(() => {
+    dispatch(fetchError());
+  });
+};
+
+export const addBookApi = (book) => (dispatch) => {
+  axios({
+    method: 'POST',
+    url: `${url}/${appId}/books/`,
+    data: {
+      item_id: book.id,
+      title: book.title,
+      author: book.author,
+      category: 'abc',
+    },
+  }).catch(() => {
+    dispatch(fetchError());
+  });
+  dispatch(getBookApi());
```

Comment:
- [ ] Please kindly ensure that the addition of book works as it should be. At the moment, when you add a book, this is the result in the browser.
- [ ] When you refresh the browser the added book disappears. Please ensure that the API call works as it should.
- [ ] Also, one can only add one book. When you try to add again, it does not work.

![image](https://user-images.githubusercontent.com/43995830/167108130-6d356ca3-16ed-4345-8e72-7b5d9ccce3e4.png)